### ISSUE TITLE
Accessibility (non-unique id) on Button + no label in SelectedRevisionTableRow

### ISSUE THREAD

**Issue by Safei-Ashraf** [mozilla/perfcompare#321]
- Close Buttons in `SelectedRevisionTableRow` have duplicated Id's => all elements got the same id "close-button" 
-  Missing label or meaningful text

**Error + Failing Tests** 
![image](https://user-images.githubusercontent.com/44810632/223851129-a97f4742-be2f-4a79-a0a4-6949e12d425e.png)

**Comment by Safei-Ashraf** [comment]
@Carla-Moz I have fixed the in issue fixed in PR #320
